include/deal.II/fe/fe_enriched.h

```diff
@@ -148,7 +148,9 @@ DEAL_II_NAMESPACE_OPEN
  @Article{Davydov2016,
    Title                    = {On the h-adaptive PUM and hp-adaptive FEM approaches applied to PDEs in quantum mechanics.},
    Author                   = {Davydov, D and Gerasimov, T and Pelteret, J.-P. and Steinmann, P.},
-   Journal                  = {Journal of Computational Physics. Submitted},
+   archivePrefix            = {arXiv},
```

Comment:
i see those, not sure which one is right. arXiv itself suggest the current solution (see link above).